### PR TITLE
Added terse notes for 2023-07-27 board meeting

### PR DESCRIPTION
**Pull request by gruetter**
This PR adds terse notes for our board meeting on 2023-07-27.

### CODE REVIEW

File: content/en/about/board/2023-07-27.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: '2023-07-27'
+---
+
+### Date and Time of Meeting
+
+2023-07-27, 2pm - 3pm CEST
+
+* Call to order: 14:06 CEST
+* Adjourned: 15:01 CEST
+* Next Meeting: 2023-08-31, 7pm - 8pm CEST
```
Comment (spier) [inline content/en/about/board/2023-07-27.md]:
```suggestion
* Next Meeting: 2023-08-31, 9am CEST
```

@gruetter I have this in the calendar as our next meeting.

Also announcing the start time should do (saving us typing and potential mistakes).

Comment (gruetter) [reply content/en/about/board/2023-07-27.md]: I was referring to[ this schedule](https://innersourcecommons.slack.com/archives/G012NHSA7DZ/p1685034149803599). Is this not the correct one, @spier ?

Comment (spier) [reply content/en/about/board/2023-07-27.md]: @gruetter The slack thread that you linked to had contains discussion of the meeting times. However the first image in that thread are NOT the times that we ended up with. We discussed various options and eventually decided on two slots, one optimized for Asia+Europe, and one for Americas+Europe.

The correct schedule is what is in the "InnerSource Commons Calendar".

Comment (spier) [reply content/en/about/board/2023-07-27.md]: @gruetter I am merging this, as I confirmed that 9am CEST is indeed the start time of the next meeting.

**Review by spier — APPROVED** [review]
LGTM

Will try to get a 2nd review in Slack to get this PR merged.

**Review by MaineC — APPROVED** [review]

**Pull request merged**